llvm/test/tools/llvm-debuginfo-analyzer/DWARF/DW_AT_GNU_template_name.test

```diff
@@ -0,0 +1,60 @@
+; REQUIRES: x86-registered-target
+
+; For the given test case:
+
+;  1 // Template Template parameters.
+;  2 // Simple template class
+;  3 template <typename T> class Foo { T Member; };
+;  4
+;  5 // Template template class
+;  6 template <template <typename T> class TemplateType>
+;  7 class Bar {
+;  8   TemplateType<int> Int;
+;  9 };
+; 10
+; 11 template <template <template <typename> class> class TemplateTemplateType>
+; 12 class Baz {
+; 13   TemplateTemplateType<Foo> Foo;
+; 14 };
+; 15
+; 16 typedef Baz<Bar> Example;
+; 17
+; 18 Example TT;
+
+; The llvm-debuginfo-analyzer did not support the DW_AT_GNU_template_name
+; attribute. When using '--attribute=encoded' and the template argument is
+; another template, it will show:
```

Comment:
```suggestion
; The llvm-debuginfo-analyzer did not support the DW_AT_GNU_template_name
; attribute. When using '--attribute=encoded' and the template argument is
; another template, it would show:
```